core/repository/airdropHistory/HistoryStorage.ts

```diff
@@ -0,0 +1,45 @@
+import { AVLTree } from 'binary-search-tree';
+import { AirdropHistory } from 'core/repository/airdropHistory/interfaces';
+import { TransactionId, Timestamp } from 'shared/model/types';
+
+type Where = {
+    rewardTime?: {
+        $qt?: Timestamp,
+        $gte?: Timestamp,
+        $lt?: Timestamp,
+        $lte?: Timestamp
+    }
+};
+
+export default class HistoryStorage {
```

Comment:
It is should be named as an AirdropHistoryStorage because it is not universal history storage